cdk/package.json

```diff
@@ -39,6 +39,7 @@
     "moment": "^2.29.3",
     "pg": "^8.7.3",
     "pg-format": "^1.0.4",
+    "process": "^0.11.10",
```

Comment:
I was getting: /Users/breuch/BlueConduit/open-data-platform/cdk/node_modules/ts-node/src/index.ts:513
    return new TSError(diagnosticText, diagnosticCodes)
           ^
TSError: ⨯ Unable to compile TypeScript:
src/open-data-platform/data-plane/api/api-stack.ts:8:21 - error TS2307: Cannot find module 'node:process' or its corresponding type declarations.

8 import { cwd } from 'node:process';
                      ~~~~~~~~~~~~~~
when I tried it originally and it seems to be due to the older version of node we are using but I remember you did that on purpose for a cdk reason? https://github.com/firebase/firebase-tools/issues/4595

Reply:
Ah, looks like Node 18 uses `node:process` but below that uses `process`. Since CodeBuild doesn't support Node 18, let's use Node 16 on our local machines to match CodeBuild. That should allow you to import from `process` without installing a 3rd party package.

Then we can define that constraint in package.json, similar to [how we do for the client](https://github.com/BlueConduit/open-data-platform/blob/main/client/package.json#L62).

Reply:
Ah thank you! That worked!